.idea/jarRepositories.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
This file is not required please remove it.

Reply:
.idea/jarRepositories.xal naa..or ALL file from .idea

Reply:
Only this file `.idea/jarRepositories`

Reply:
ok

Reply:
Thankyou for your contribution : )

Reply:
Happy to contribute here :)